irods/manager/data_object_manager.py

```diff
@@ -459,8 +470,28 @@ def parallel_put(
             updatables=updatables,
         )
 
-    def create(self, path, resource=None, force=False, **options):
-        options[kw.DATA_TYPE_KW] = "generic"
+    @staticmethod
+    def _call_thru(c):
+        return c() if callable(c) else c
+
+    def create(
+        self,
+        path,
+        resource=None,
+        force=client_config.getter("data_objects", "force_create_by_default"),
```

Comment:
Is it better to use `None` as a default and then call `client_config.getter` when `create()` is invoked? My understanding is that this is evaluated at import time, which might lead to problems. I could be wrong, though.

Reply:
Okay, gotcha. Resolving.